src/main/java/hudson/plugins/jira/JiraCreateIssueNotifier.java

```diff
@@ -274,15 +282,10 @@ private JiraSession getJiraSession(AbstractBuild<?, ?> build) throws IOException
     /**
      * @param filename
      */
-    private void deleteFile(String filename, TaskListener listener) {
+    private void deleteFile(String filename) {
```

Comment:
Also I'm pretty sure that writing to the file is not the best way to preserve information between builds.. any hints this could be done in better way?